### PR TITLE
Improve dashboard filtering page

### LINKED ISSUE #2167: Improve dashboard response filtering docs
### Description
Add more examples and explanation to dashboard response filtering doc.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [x] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/dashboard/filtering/

### Expected behavior
Dashboard response filtering examples cover the most common use cases and syntax differences from API response filtering.

### Current behavior
Doc only contains examples for label-selector and filter-selector. 

### Corrections needed
Consider adding examples for:
- [ ] As a User, I want to filter Events by Entity
- [ ] As a User, I want to filter Events by Check
- [ ] As a User, I want to filter Events by Silenced
- [ ] As a User, I want to filter Events by Severity (ok, warning, critical, unknown)
- [ ] As a User, I want a list of Events that identify an incident (check state failing -- anything that is not 0)
- [ ] As a User, I want to filter Events, Entities, and Checks by one or more subscriptions

Also consider adding information about dashboard response filtering syntax differences from API response filtering syntax (possibly just by acknowledging that it is different).

From Simon: Format is specific to the client you’re using. With sensuctl, you can use both formats
`$ s event list --field-selector='linux notin event.entity.subscriptions'`
`$ s event list --field-selector 'linux notin event.entity.subscriptions'`
In the web ui, you must do `fieldSelector:linux notin event.entity.subscriptions`

### Additional context
Consider modifying examples added to API overview in https://github.com/sensu/sensu-docs/pull/2160.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds explanation and examples to dashboard filtering page.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2167

### CODE REVIEW

**Comment by hillaryfraley** [comment]
Once we merge this, I will create an issue in the web UI repo to change the link that displays in the filter bar to https://docs.sensu.io/sensu-go/latest/dashboard/filtering/.
dashboard-filtering-link

File: content/sensu-go/5.18/dashboard/filtering.md [hunk 1]
```diff
@@ -10,51 +10,160 @@ menu:
...
 
+- [Basic filters](#basic-filters)
+- [Advanced filters](#advanced-filters)
+- [Create dashboard filters](#create-dashboard-filters)
+- [Operators quick reference](#operators-quick-reference)
+- [Examples](#examples)
+
 The Sensu dashboard includes basic filters you can use to build customized views of your Sensu resources.
-When applied, filters create a unique link, so you can bookmark and share your favorite filter combinations.
+Sensu also supports advanced dashboard filtering based on a wider range of resource attributes and custom lables as a [commercial feature][1].
```
Comment (cwjohnston) [inline content/sensu-go/5.18/dashboard/filtering.md]:
```suggestion
Sensu also supports advanced dashboard filtering based on a wider range of resource attributes and custom labels as a [commercial feature][1].
```

File: content/sensu-go/5.18/dashboard/filtering.md [hunk 2]
```diff
@@ -10,51 +10,160 @@ menu:
...
+To filter resources based on [label selectors][6] or [field selectors][2], you'll write a brief filter statement.
+The standard dashboard filtering syntax is:
+
+{{< highlight text >}}
+SELECTOR:FILTER_STATEMENT
+{{< /highlight >}}
+
+To write a dashboard filter command:
+
+- Replace `ATTRIBUTE` with the selector you want to use: `labelSelector` or `fieldSelector`.
```
Comment (cwjohnston) [inline content/sensu-go/5.18/dashboard/filtering.md]:
```suggestion
- Replace `SELECTOR` with the selector you want to use: `labelSelector` or `fieldSelector`.
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Two small changes suggested, otherwise looking good!

File: content/sensu-go/5.18/dashboard/filtering.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.18/dashboard/filtering.md]: Gah thank you. That word is my Waterloo.

**Pull request merged**